highfidelity/hifi
# Home Fishtank

This PR adds a fishtank for the Home space.

To test:
1. Run this script https://rawgit.com/imgntn/hifi/newfishes/unpublishedScripts/DomainContent/Home/fishTank/createFishTank.js
2. Observe the fish swimming around inside of the tank
3. They will swim toward where you're looking when you're close to the tank.

(Note: cleanup is on by default right now -- when this is integrated into some kind of home spawner/reset script, it will persist)
